app/src/main/java/com/google/android/apps/youtube/app/settings/videoquality/VideoQualitySettingsActivity.java

```diff
@@ -0,0 +1,118 @@
+package com.google.android.apps.youtube.app.settings.videoquality;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import app.revanced.integrations.settingsmenu.ReVancedSettingsFragment;
+import app.revanced.integrations.settingsmenu.ReturnYouTubeDislikeSettingsFragment;
+import app.revanced.integrations.settingsmenu.SponsorBlockSettingsFragment;
+import app.revanced.integrations.utils.LogHelper;
+import app.revanced.integrations.utils.ReVancedUtils;
+import app.revanced.integrations.utils.ThemeHelper;
+
+// Hook a dummy Activity to make the Back button work
+public class VideoQualitySettingsActivity extends Activity {
```

Comment:
Why has this class been renamed to `VideoQualitySettingsActivity`?

Reply:
Inotia00's explanation was added to the beginning of the PR.

Reply:
That is not an explanation why it is renamed to `VideoQualitySettingsActivity`.

Reply:
> That is not an explanation why it is renamed to `VideoQualitySettingsActivity`.

I think he wanted to make explicit the name of the activity hooked by settings. This is my hypothesis.

But yes, this should be restored to the old name.